config.c

```diff
@@ -414,7 +436,7 @@ int parse_config_arguments(struct mako_config *config, int argc, char **argv) {
 
 		const char *name = long_options[option_index].name;
 		if (!apply_style_option(&global_criteria(config)->style, name, optarg)
-				|| apply_config_option(config, name, optarg)) {
+				&& !apply_config_option(config, name, optarg)) {
```

Comment:
This looked wrong to me, and I was  getting a parse failure for `--output` until I changed it to this.

Reply:
Oh right, good catch